README.md

```diff
@@ -5,6 +5,8 @@ Transifex Command-Line Tool
 [![codecov](https://codecov.io/gh/transifex/transifex-client/branch/master/graph/badge.svg)](https://codecov.io/gh/transifex/transifex-client)
 [![PyPI version](https://badge.fury.io/py/transifex-client.svg)](https://badge.fury.io/py/transifex-client)
 
+[Learn how to use the Transifex CLI through an interactive tutorial](https://www.transifex.com/learn/txclient/)
```

Comment:
_Suggestion:_ Maybe we could create a new section called `Usage` and add
```
Click here for complete documentation on the Transifex Command-line Tool via our documentation site or learn how to use it through an interactive tutorial.
```
with a link to the `interactive tutorial` part.

The "Click here..." is in the `Description` section right now.
@rigaspapas @mikegianno @nelefth

Reply:
Suggestion #1:
New section after Description called "Getting started with TX client"

Suggestion #2:
Whether you have experience with the Linux command line or not, this interactive tutorial is intended for everyone who wishes to learn how Transifex Client works. There is no need to download anything - Just click on the link provided above, and follow the instructions. Good luck!

For more information about TX client, please visit our documentation guide here.
@rigaspapas  @codegaze  @mikegianno

Reply:
I updated this PR using Nina's suggestions. Can you please check again?

Reply:
+1. 
I would only omit the `Good Luck` part.

Reply:
It looks good :)